packages/menu/menu-item.md

```diff
@@ -159,3 +204,83 @@ When the `value` attribute is leveraged, the selected `<sp-menu-item>` represent
     </sp-menu-item>
 </sp-action-menu>
 ```
+
+</sp-tab-panel>
+</sp-tabs>
+
+#### Submenu
+
+An `<sp-menu-item>` can also accept content addressed to its `submenu` slot. Using the `<sp-menu>` element with this slot name the options will be surfaced in flyout menu that can be activated by hovering over the root menu item with your pointer or focusing the menu item and pressing the appropriate `ArrowRight` or `ArrowLeft` key based on text direction to move into the submenu.
```

Comment:
```suggestion
An `<sp-menu-item>` can also accept content addressed to its `"submenu"` slot. An `<sp-menu>` element with this slot name surfaces the options in an adjacent popover, which can be activated by hovering over the parent menu item with your pointer or focusing the menu item and pressing the appropriate `ArrowRight` or `ArrowLeft` key based on text direction to move into the submenu.
```
I tried to match the [language on our Spectrum docs site for Menu](https://spectrum.corp.adobe.com/page/menu/#Popover-submenus) here. We could also say "triggering menu item" instead of "parent"?